src/integration/resources/coordinator_client.go

```diff
@@ -881,3 +882,80 @@ func toResponse(
 
 	return nil
 }
+
+// GraphiteQuery retrieves graphite raw data.
+func (c *CoordinatorClient) GraphiteQuery(
+	graphiteReq GraphiteQueryRequest,
+) ([]Datapoint, error) {
+	if graphiteReq.From.IsZero() {
+		graphiteReq.From = time.Now().Add(-24 * time.Hour)
```

Comment:
Is this what graphite does by default (i.e. make the query window 24h if not specified)?

Reply:
yes